components/Starknet/modules/architecture_and_concepts/pages/Accounts/validate_and_execute.adoc

```diff
@@ -1,55 +1,80 @@
 [id="validate_and_execute"]
-= Validate and execute
+= The `+__validate__+` and `+__execute__+` functions
 
-As previously stated, Starknet's account structure is not completely arbitrary, but must include the following two functions, which account transactions call:
+A Starknet account is a contract that must include the following two functions:
 
-* `&lowbar;&lowbar;validate&lowbar;&lowbar;`
-* `&lowbar;&lowbar;execute&lowbar;&lowbar;`
+[horizontal,labelwidth=15]
+`+__validate__+`:: Initiates the validation stage in the sequencer.
+`+__execute__+`:: Initiates the execution stage in the sequencer.
 
-Separating the `&lowbar;&lowbar;validate&lowbar;&lowbar;` and `&lowbar;&lowbar;execute&lowbar;&lowbar;` functions guarantees payment to sequencers for work completed and protects them from Denial of Service (DoS) attacks.
+When the sequencer receives a transaction request, it calls the `+__validate__+` function with the fields of the transaction request. After the validation stage completes successfully, the sequencer calls the `+__execute__+` function with the fields of the transaction request.
+
+* For more information on the available transaction types and their fields, see xref:architecture_and_concepts:Network_Architecture/transactions.adoc[Transaction types].
+* For more information on the validation and execution stages, see xref:architecture_and_concepts:Network_Architecture/transaction-life-cycle.adoc[Transaction lifecycle].
+
+Separating the validation and execution stages, which correspond to the `+__validate__+` and `+__execute__+` functions, guarantees payment to sequencers for work completed and protects them from Denial of Service (DoS) attacks.
 
 [id="the_validate_function"]
-== The validate function
+=== The `+__validate__+` function
 
-The `&lowbar;&lowbar;validate&lowbar;&lowbar;` function ensures that any transaction submitted was indeed initiated by the account owner and therefore will not take up unjustified resources during the execution process.
+The `+__validate__+` function ensures that any transaction submitted was indeed initiated by the account owner and therefore does not take up unjustified resources during the execution process.
```

Comment:
`validate` can have arbitrary logic, usually it checks the signature, but it's not mandatory at the protocol level.

Reply:
@odednaor What _is_ mandatory at the protocol level?